pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml

```diff
@@ -29,6 +29,27 @@ spec:
         heritage: {{ .Release.Service }}
     spec:
       containers:
+      {{- if and .Values.prometheus.enabled (eq .Values.kubernetesDistribution "OpenShift") }}
```

Comment:
I thought about suggesting we make this a separate value, but it seems like there would be too many fiddly bits to configure, especially the certificate? What do you think?

Reply:
Do you mean a separate value to turn on kube-rbac-proxy on without OpenShift?

I think it's a good idea but I think it makes it too complex so I'd like to avoid it unless someone from the community requests it.